scripts/deploy_minio_on_kind.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
@tarilabs @syntaxsdev one question I have is that I was assuming we are using pytest libraries for any cluster management too, is that a wrong assumption?

Reply:
no we are not "orchestrating cluster setup from pytest" if that was the question, as the request was to stick to makefile and scripting; the downside is netting on a basic common ground, but I see the advantage as reuse from Go or else in the future if needed.

Reply:
Correct for reuse and manageability, this was the topic in shift left.  @lugi0 might have some comments on this. I am not asking to change this PR but maybe we can capture a follow-up if that is the overall direction we need to be writing the tests with, if not we can ignore.

Reply:
I'm favourable of revising the setup, as you know I prsonally prefer to reduce Makefile and scripts to the minimum as possible, but we need indeed to account for multiple factors.